ansible/roles/ocp4-workload-mlops/tasks/install-jupyterhub.yaml

```diff
@@ -4,11 +4,14 @@
     state: present    
     definition: "{{ lookup('file', 'jupyterhub_kfdef.yaml' ) | from_yaml }}"
 
-- name: Add repo for jupyterhub users
-  include_tasks: add-git-repo-jupyterhub.yaml
-  loop: "{{users}}"
-  loop_control:
-    loop_var: user
-
-
+- name: "Wait for jupyterhub pod to be in Ready state"
+  k8s_info:
+    kind: Pod
+    label_selectors:
+      - 'app=jupyterhub'
+    namespace: labs-infra
+  register: pod_status
```

Comment:
It would be a good practice for you to standardize on registers like this. We typically recommend that they are something like `r_whatever`. It makes it easier to narrow down the scope and location of a variable that might accidentally get reused later.